docs/install-sample-packs.md

```diff
@@ -88,11 +88,11 @@ No. These sample data packs are only compatible with the Microsoft 365 developer
 
 ## How can I see the sample data in my subscription?
 
-After you install the Users sample data pack, to see the users that were added, go to the **Microsoft 365 Admin Center** on your Microsoft 365 developer subscription. Under **Users**, select **Active users**. You will see the list of 16 users. You can select a user to view the associated metadata, including photos and licenses.
+After you install the Users sample data pack, to see the users that were added, go to the [**Microsoft 365 Admin Center**](https://admin.microsoft.com/) on your Microsoft 365 developer subscription. Under **Users**, select **Active users**. You will see the list of 16 users. You can select a user to view the associated metadata, including photos and licenses.
```

Comment:
```suggestion
After you install the Users sample data pack, to see the users that were added, go to the [**Microsoft 365 Admin Center**](https://docs.microsoft.com/en-us/microsoft-365/admin/admin-overview/about-the-admin-center?view=o365-worldwide) on your Microsoft 365 developer subscription. Under **Users**, select **Active users**. You will see the list of 16 users. You can select a user to view the associated metadata, including photos and licenses.
```
Changed link to the docs help page. Not everyone has access to the admin center so this is the intermediary step to sign up.